src/ResizeCallbackMaker.js

```diff
@@ -134,8 +134,13 @@ ResizeCallbackMaker.prototype.startResizingCallbacks = function(iframe, callback
 			cleanup: stopListening
 		};
 	} else {
+		// Hack to prevent the callback from being repeatedly called when rendering a file in Lessons.
+		// This was causing height & overflow-y to be incorrect as a recent
+		// change caused these to now be on the same domain.
+		const iframeIsLessonsFile = iframe && iframe.classList.contains('html-topic-iframe');
+
 		var props = {
-			toggle: true,
+			toggle: !iframeIsLessonsFile,
```

Comment:
this toggle property acts as an exit early in the callback for this function. I don't think we need to keep calling it in this case.
https://github.com/Brightspace/react-valence-ui-iframe/blob/d1cc52457a066936ec43f4101129649e7ee42a1c/src/ResizeCallbackMaker.js#L14